sql/moz-fx-data-shared-prod/firefox_desktop_derived/cfr_users_last_seen_v2/metadata.yaml

```
@@ -0,0 +1,18 @@
+friendly_name: Messaging System Cfr Users Last Seen
+description: |-
+      Captures history of activity of each client using CFR in 28 day windows for each submission date.
+owners:
+- gleonard@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name: bqetl_messaging_system
+bigquery:
+  time_partitioning:
+    type: day
+    field: 'submission_date'
```

```suggestion
    field: submission_date
```

single quotes are redundant here.